packages/ic-asset-certification/README.md

```diff
@@ -749,3 +749,22 @@ use ic_cdk::api::set_certified_data;
 
 set_certified_data(&asset_router.root_hash());
 ```
+
+## Querying assets
+
+The `AssetRouter` has two functions to retrieve an `AssetMap` containing assets.
+
+The `get_assets()` function returns all standard assets, while the `get_fallback_assets()` function returns all fallback assets.
+
+The `AssetMap` can be used to query assets by `path`, `encoding`, and `starting_range`.
+
+For standard assets, the path refers to the asset's path, e.g. `/index.html`.
+
+For fallback assets, the path refers to the scope that the fallback is valid for, e.g. `/`. See the `fallback_for` config option for more information on fallback scopes.
+
+For all types of assets, the encoding refers to the encoding of the asset, see `AssetEncoding`.
+
+Assets greater than 2mb are split into multiple ranges, the starting range allows retrieval of
+individual chunks of these large assets. The first range is `Some(0)`, the second range is
+`Some(2_000_000)`, the third range is `Some(4_000_000)`, and so on. The entire asset can
```

Comment:
nit: maybe  make the constat `ASSET_CHUNK_SIZE` public and use it here instead of the numbers?

Reply:
Good point, thanks